astrosat/utils/utils_logging.py

```diff
@@ -49,7 +49,7 @@ def emit(self, record):
                 lambda x: x[0],
                 [
                     DatabaseLogTag.objects.get_or_create(name=tag_name)
-                    for tag_name in getattr(record, "tags", [])
+                    for tag_name in getattr(record, "tags", []) or []
```

Comment:
If "tags" was not set this would have returned `[]`.  But if it _was_ set to `None`, this would have returned `None`.  This code ensures I return `[]` in that case too.